src/Futhark/CodeGen/Backends/CCUDA.hs

```diff
@@ -134,6 +134,13 @@ cliOptions =
              optionArgument = NoArgument,
              optionDescription = "Gather profiling data while executing and print out a summary at the end.",
              optionAction = [C.cstm|futhark_context_config_set_profiling(cfg, 1);|]
+           },
+         Option
+           { optionLongName = "use-single-device",
```

Comment:
This is fine for development, but we'll need to think about what the configuration API should actually be like.  It's fiddly stuff that isn't conceptually very interesting or difficult, but can still be very time-consuming.  I think Cosmin and I will try to work something out.